config/secrets.yml

```diff
@@ -130,9 +130,9 @@ test:
   should_send_notifications: true
   # Sendgrid
   # a real api_key is needed for recording new VCR based tests
-  sendgrid_api_offline: true
+  sendgrid_api_offline: false
```

Comment:
This should remain true unless we update all the tests which also trigger the sendgrid updates.